src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/ComIntegrationTest.kt

```diff
@@ -51,19 +60,71 @@ class ComIntegrationTest : IntegrationTestBase() {
     assertThat(result?.developerMessage).isEqualTo("This is a message")
   }
 
+  @Test
+  fun `Search for offenders on a staff member's caseload`() {
+    probationSearchApiMockServer.stubPostLicenceCaseloadByTeam(Gson().toJson(aLicenceCaseloadSearchRequest))
+
+    val result = webTestClient.post()
+      .uri("$probationSearchApiWiremockUrl/licence-caseload/by-team")
```

Comment:
Ah didn't spot this last time but this test is solely testing a test web client can hit the test version of wiremock works as you'd expect wiremock to. 

This should hit your new endpoint, like this example:
```
  fun `Get a licence by ID`() {
    val result = webTestClient.get()
      .uri("/licence/id/1")
      .accept(MediaType.APPLICATION_JSON)
      .headers(setAuthorisation(roles = listOf("ROLE_CVL_ADMIN")))
      .exchange()
      .expectStatus().isOk
      .expectHeader().contentType(MediaType.APPLICATION_JSON)
      .expectBody(Licence::class.java)
      .returnResult().responseBody

    log.info("Expect OK: Licence is ${mapper.writeValueAsString(result)}")

    assertThat(result?.standardLicenceConditions?.size).isEqualTo(2)
    assertThat(result?.standardLicenceConditions)
      .extracting("code")
      .containsAll(listOf("goodBehaviour", "notBreakLaw"))
    assertThat(result?.standardPssConditions?.size).isEqualTo(1)
    assertThat(result?.standardPssConditions)
      .extracting("code")
      .containsAll(listOf("attendMeetings"))
  }
```